gslib/tests/util.py

```diff
@@ -623,14 +623,19 @@ def MakeBucketNameValid(name):
   """Returns a copy of the given name with any invalid characters replaced.
 
   Args:
-    name (str): The bucket name to transform into a valid name.
+    name (str, unicode, bytes): The bucket name to transform into a valid name.
 
   Returns:
-    (str) The version of the bucket name containing only valid characters.
+    (str, unicode, bytes) The version of the bucket name containing only
```

Comment:
Nit, here and in the line above: Can you make these "(str, unicode, or bytes)" or "Union[str, unicode, bytes]"? The versions without the "or" sort of makes it look like a tuple.

Reply:
Good point, I will change it.